File: SVAuto.EF/Model/Order.cs
using System;

namespace SVAuto.EF.Model
{
    public class Order : BaseEntity
    {
        public string Client { get; set; }
        public string Phone { get; set; }
        public string Car { get; set; }
        public string Part { get; set; }
        public DateTimeOffset CreationDateTime { get; set; }
        public DateTimeOffset ModificationDateTime { get; set; }

        public int StatusId { get; set; }
        public virtual OrderStatus Status { get; set; }
    }
}

File: clients/SVAuto-Web/src/app/modules/simple-store/models/order.ts
import { OrderStatus } from "./order-status";

export class Order {
  public id: number;
  public client: string;
  public phone: string;
  public car: string;
  public part: string;
  public cost: number;
  public creationDate: Date;
  public modificationDate: Date;

  public status: OrderStatus;
}
